src/material/chips/chip-row.ts

```diff
@@ -69,7 +69,7 @@ export interface MatChipEditedEvent extends MatChipEvent {
     '[attr.aria-description]': 'null',
     '[attr.role]': 'role',
     '(focus)': '_handleFocus($event)',
-    '(dblclick)': '_handleDoubleclick($event)',
+    '(click)': '_handleClick($event)',
```

Comment:
I know that this is what #27106 calls for, but @zarend I'm not sure if it's the correct thing to do here. With this approach users don't have a way of setting focus on the chips using the mouse without getting the chip into the editable state. The original issue mentions "Double click is generally not available to Screen Readers and Assistive Technology.", but users can still get into the editable state by pressing enter.

This can also be problematic if the users project other clickable content into the chip.

Reply:
Hello, I'm still researching https://github.com/angular/components/issues/27106 to determine what our options are.